instrumentation-core/spring/spring-webmvc-3.1-core/src/main/java/io/opentelemetry/instrumentation/springwebmvc/HandlerMappingResourceNameFilter.java

```diff
@@ -14,10 +14,10 @@
  * limitations under the License.
  */
 
-package io.opentelemetry.auto.instrumentation.springwebmvc;
+package io.opentelemetry.instrumentation.springwebmvc;
 
 import static io.opentelemetry.auto.bootstrap.instrumentation.decorator.HttpServerTracer.CONTEXT_ATTRIBUTE;
-import static io.opentelemetry.auto.instrumentation.springwebmvc.SpringWebMvcDecorator.DECORATE;
+import static io.opentelemetry.instrumentation.springwebmvc.SpringWebMvcDecorator.DECORATE;
 
 import io.grpc.Context;
```

Comment:
just realized OT is still using gRPC Context?

Reply:
Almost all-in :) There are some discussions, but for now it is everywhere.